test/parallel/test-global-customevent.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --experimental-global-customevent --expose-internals
+'use strict';
+
+const common = require('../common');
+const { strictEqual, ok } = require('node:assert');
+const { Worker, isMainThread } = require('worker_threads');
+const { CustomEvent } = require('internal/event_target');
+
+ok(CustomEvent);
+strictEqual(globalThis.CustomEvent, CustomEvent);
```

Comment:
I think you meant the test the global, right?
```suggestion
ok(CustomEvent);

{
  const { CustomEvent } = require('internal/event_target');
  strictEqual(globalThis.CustomEvent, CustomEvent);
}
```
or
```suggestion
const { CustomEvent: internalCustomEvent } = require('internal/event_target');

ok(CustomEvent);
strictEqual(CustomEvent, internalCustomEvent);
```

Reply:
Applied.